pvlib/pvsystem.py

```diff
@@ -1133,6 +1121,28 @@ def calcparams_desoto(effective_irradiance, temp_cell,
          Source: [4]
     '''
 
+    # test for use of function pre-v0.6.0 API change
+    if isinstance(a_ref, dict) or \
+       (isinstance(a_ref, pd.Series) and ('a_ref' in a_ref.keys())):
+        import warnings
+        warnings.warn('module_parameters detected as fourth positional'
```

Comment:
Recommend using ``warnings.warn(message, DeprecationWarning)`` where message is both of your statements as one.

Reply:
Done